Is it possible to use this with lazy_static?

Can't get it compile.

```
error[E0277]: the trait bound `*mut onig_sys::OnigRegexType: std::marker::Sync` is not satisfied in `onig::Regex`
  --> src/lib.rs:14:1
   |
14 | / lazy_static! {
15 | |     static ref ROLES_FIND_RE: Regex = {
16 | |         let mut roles: Vec<String> = include_str!("../litigant/data/roles.txt")
17 | |             .lines()
...  |
52 | |     static ref ENDSWITH_COMPANY_RE: Regex = Regex::new("company$").unwrap();
53 | | }
   | |_^ `*mut onig_sys::OnigRegexType` cannot be shared between threads safely
   |
   = help: within `onig::Regex`, the trait `std::marker::Sync` is not implemented for `*mut onig_sys::OnigRegexType`
   = note: required because it appears within the type `onig::Regex`
   = note: required by `lazy_static::lazy::Lazy`
   = note: this error originates in a macro outside of the current crate
```

onig version: 1.4.1

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -138,6 +138,9 @@ pub struct Regex {
     raw: onig_sys::OnigRegexMut,
 }
 
+unsafe impl Send for Regex {}
+unsafe impl Sync for Regex {}
+
 impl Error {
     fn new(code: c_int, info: onig_sys::OnigErrorInfo) -> Error {
         let mut buff = &mut [0; onig_sys::ONIG_MAX_ERROR_MESSAGE_LEN as usize];

